_rules/7d6734.md

```diff
@@ -0,0 +1,207 @@
+---
+id: 7d6734
+name: SVG image has accessible name
+rule_type: atomic
+description: |
+   This rule checks that each SVG image element that is included in the accessibility tree has an accessible name.
+accessibility_requirements:
+  wcag20:1.1.1: # Non-Text Content (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Bryn Anderson
+---
+
+## Applicability
+
+The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with a [semantic role](#semantic-role) of either `img`, `graphics-document`, `graphics-object`, `graphics-symbol`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with a [semantic role](#semantic-role) of either `img`, `graphics-document`, `graphics-object`, `graphics-symbol`, that is [visible](#visible), and [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+**Note**: The [SVG Accessibility API Mappings](https://www.w3.org/TR/svg-aam-1.0/#include_elements) specifies that many elements in the SVG namespace are purely presentational and should not be included in the accessibility tree unless indicated otherwise through the use of text alternative content, an explicit WAI ARIA role, or a valid `tabindex` attribute.
+
+## Expectation
+
+Each target element has an [accessible name](#accessible-name) that is not [whitespace](#whitespace).
```

Comment:
"not whitespace" => "not *only" whitespaces"

Reply:
Not sure what the difference is here but fair enough.

Reply:
At least for consistency with how it,s written in the examples :-D